### DISCUSSION TITLE
Filtering problem with space

### DISCUSSION THREAD

**Discussion by KCLeeRicky** [Snow-Shell/servicenow-powershell#259] [Q&A]
I put this filter @('hardware_status','-like','In use') and nothing returned.
Changed it before space and a record returned @('hardware_status','-like','In').
How I can filter it by 'In use'? Please

**Comment by gdbarron** [comment]
Hi @KCLeeRicky.  I am not able to repro this unfortunately.  Perhaps try `-eq` instead of `-like` if you want to compare the entire value?
